hack/update_submodules.sh

```
@@ -32,11 +32,12 @@ function update_libraries() {
   go get github.com/openshift/api@$base_branch
   go get github.com/openshift/client-go@$base_branch
   go get github.com/openshift/library-go@$base_branch
-  go get github.com/openshift/controller-runtime-common@$base_branch
-  # react to main branch change in machine-config-operator
+  # react to main branch change in controller-runtime-common and machine-config-operator
   if [ "$base_branch" == "master" ]; then
+    go get github.com/openshift/controller-runtime-common@main
     go get github.com/openshift/machine-config-operator@main
    else
+    go get github.com/openshift/controller-runtime-common@$base_branch
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)